bin/ard_pbs

```diff
@@ -212,19 +293,25 @@ def _submit_summary(indir, outdir, batch_id, pbs_resources, env, job_ids, test):
               help="Job walltime in `hh:mm:ss` format.")
 @click.option("--email", default="",
               help="Notification email address.")
+@click.option("--datacube-env", type=click.Path(exists=True, readable=True),
```

Comment:
Consider if a default option of None could work here.  I assume this is needed for indexing.  Since indexing is optional it would be useful to have --datacube-env optional too.

Reply:
Ok thanks I'll look into that.  Essentially they're optional, but if one is set, both have to be set.

Reply:
A cleaner way might be to only have the `--datacube-env` option, and if set it is then implicit that indexing is to occur.
This would remove one command switch, and avoid the additional complexity of dealing with two optional but dependent switches.

What do you think?